VAS.Core/Common/RangeObservableCollection.cs

```
@@ -25,21 +24,35 @@ namespace VAS.Core.Common
 {
 	public class RangeObservableCollection<T> : ObservableCollection<T>
 	{
+		public RangeObservableCollection () : base ()
+		{
+		}
+
+		public RangeObservableCollection (List<T> list) : base (list)
+		{
+		}
+
+		public RangeObservableCollection (IEnumerable<T> collection) : base (collection)
+		{
+		}
+
 		public void AddRange (IEnumerable<T> items)
 		{
-			foreach (var item in items) {
-				Items.Add (item);
+			if (items.Any ()) {
```

I don't know if we should check for null. Maybe better to let the app crash, because shouldn't be possible to do AddRange(null)